torch/functional.py

```diff
@@ -290,6 +290,7 @@ def meshgrid(*tensors, **kwargs):
     return torch._C._VariableFunctions.meshgrid(tensors)
 
 
+_VF_stft = torch._C._VariableFunctions.stft
```

Comment:
Can you make this use [`_VF.py`](https://github.com/pytorch/pytorch/blob/1969898647bf33c0b1fd2625c02075d2d543f9e2/torch/nn/_VF.py#L6)? You'll probably have to move that file somewhere else but it will make it so we can just do the below instead of imports for every function

https://github.com/pytorch/pytorch/blob/cfb4862673303a2725d815c0338435f6b6c699fb/torch/nn/functional.py#L13